assets/js/googlesitekit/datastore/user/key-metrics.test.js

```diff
@@ -280,7 +280,7 @@ describe( 'core/user key metrics', () => {
 					.dispatch( CORE_USER )
```

Comment:
Tests updated due to the `includeConversionTailoredMetrics` setting type change, and expanded to verify that ACR metrics are showing only when detected events needed for them to appear are present